pages/modules/[module].tsx

```diff
@@ -519,6 +492,23 @@ const ModulePage: NextPage<ModulePageProps> = ({
             </div>
           </div>
         </div>
+        {versionInfo.stardocs.length > 0 && (
+          <div className="max-w-7xl w-7xl mx-auto mt-8">
+            <div className="mt-6">
+              <h2 className="text-2xl font-bold mb-4">
+                Starlark API Documentation
+              </h2>
+              <div className="space-y-4">
+                {versionInfo.stardocs.map((stardoc, index) => (
+                  <StardocRenderer
```

Comment:
We probably want to render these using Bazel, as this won't scale past 10 modules.

Reply:
I'll look into this